cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -472,7 +472,7 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 			}
 
 			bucket := targetOptions.Bucket
```

Comment:
I think that a nice addition would be erroring out if the bucket name isn't set in either the config, nor target options.

Reply:
Sounds reasonable....